### PR TITLE
Implement basic Flight Physics

### PR DESCRIPTION
**Pull request by widdrr**
Implemented basic Flight Physics based on Thrust, Drag and Lift.

- Thrust is self explanatory from the code.
- Lift is computed from the velocity, angle of attack and fine-tuning parameters and is computed for both the wings, and for the rudder (you could call it a horizontal "lift")
- Induced Drag is computed from lift.
- Parasitic and Angular Drag mainly depend on speed

PR also contains some testing input code and a test scene (They will be replaced with proper implementations later)

### CODE REVIEW

**Review by rob3rtu — APPROVED** [review]
I trust your work, i saw the recordings. Good job! 😎

**Comment by widdrr** [comment]
That's not how this works...

**Review by AdelaCorbeanu — APPROVED** [review]
great job :)

File: Flight Game/Assets/Scripts/Plane Pshysics/FlightPhysics.cs [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+    [Tooltip("Parameter that affects 'Lift' generaed by the rudder")]
+    [SerializeField]
+    private float _rudderPower;
+
+    [Tooltip("Parameter that affects Induced Drag, a type of Drag caused by Lift")]
+    [SerializeField]
+    private float _inducedDragFactor;
+
+    [Tooltip("Coefficinets for Angular Drag in the 3 main axes")]
```
[the remark this answers is not in the document]
Comment (widdrr) [reply Flight Game/Assets/Scripts/Plane Pshysics/FlightPhysics.cs]: already fixed

File: Flight Game/Assets/Scripts/Plane Pshysics/CameraRestrict.cs
[the remark this answers is not in the document]
Comment (widdrr) [reply Flight Game/Assets/Scripts/Plane Pshysics/CameraRestrict.cs:1]: already fixed

File: Flight Game/Assets/Scripts/Plane Pshysics/FlightPhysics.cs [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+    [Range(0,1)]
+    [Tooltip("The fraction of the Maximum Thrust currently applied")]
+    [SerializeField]
+    private float _throttle;
+
+    [Tooltip("Parameter that affects Lift generated by the wings")]
+    [SerializeField]
+    private float _wingPower;
+
+    [Tooltip("Parameter that affects 'Lift' generaed by the rudder")]
```
[the remark this answers is not in the document]
Comment (widdrr) [reply Flight Game/Assets/Scripts/Plane Pshysics/FlightPhysics.cs]: Technically, both forces are types of Lift, but one generally equates Lift with an upwards force. The "Lift" generated by the rudder acts horizontally. I put it in quotes to emphasize this aspect.

**Pull request merged**